erdpy/accounts.py

```diff
@@ -59,13 +64,19 @@ def __init__(self, address: Any = None, pem_file: Union[str, None] = None, pem_i
             address_from_key_file, seed = load_from_key_file(key_file, password)
             self.private_key_seed = seed.hex()
             self.address = Address(address_from_key_file)
+        elif ledger:
+            from erdpy.ledger.ledger_functions import do_get_ledger_address
```

Comment:
also maybe you can extract this code in a new method ---> the constructor of this class gets bigger and bigger

Reply:
no need. it's just a function call and a variable assignment